docs: description client-control plugin configuration `max_body_size` unit (#10635)
apache/apisix: docs/en/latest/plugins/client-control.md

| Name          | Type    | Required | Valid values | Description                                                                                                                          |
| ------------- | ------- | -------- | ------------ | ------------------------------------------------------------------------------------------------------------------------------------ |

> | max_body_size | integer | False    | [0,...]      | Set the maximum limit for the client request body and dynamically adjust the size of [`client_max_body_size`](https://nginx.org/en/docs/http/ngx_http_core_module.html#client_max_body_size), measured in bytes. |

Reviewer: - The unit of the [variable](https://nginx.org/en/docs/http/ngx_http_core_module.html#client_max_body_size) is `m`, why `max_body_size` is `bytes` here?
- What does it mean to set `max_body_size` to 0?
- The corresponding code also needs to be commented.

Author: Hi @moonming 
For a description of the units check this document https://nginx.org/en/docs/syntax.html, in https://nginx.org/en/docs/http/ngx_http_core_module.html#client_max_body_ The unit of `client_max_body_size` is not specified in size, but the default value is `1m`. If you configure `client_max_body_size 1024` in the configuration file, then the maximum request body size is 1024 bytes.

Author: > What does it mean to set max_body_size to 0?
> The corresponding code also needs to be commented.

That's a great suggestion.

Reviewer 2: [`Content-Length`](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Length) is in bytes and is used in comparison with the configured `max_body_size` here:

https://github.com/apache/apisix/blob/4db435ee38b425da6364ad41aa9c019046190d87/apisix/plugins/client-control.lua#L57

> | max_body_size | integer | False    | [0,...]      | Set the maximum limit for the client request body and dynamically adjust the size of [`client_max_body_size`](https://nginx.org/en/docs/http/ngx_http_core_module.html#client_max_body_size), measured in bytes. When `client_max_body_size` is set to 0, no check will be performed on the client request body size. |

Reviewer: ```suggestion
| max_body_size | integer | False    | [0,...]      | Set the maximum limit for the client request body and dynamically adjust the size of [`client_max_body_size`](https://nginx.org/en/docs/http/ngx_http_core_module.html#client_max_body_size), measured in bytes. If you set the `max_body_size` to 0, then the size of the client's request body will not be checked. |
```
